eoscompanion/dummy_feed.py

```diff
@@ -0,0 +1,117 @@
+# /eoscompanion/dummy_feed.py
+#
+# Copyright (C) 2018 Endless Mobile, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+# All rights reserved.
+'''Function to provide a dummy feed.'''
+
+def dummy_feed():
+    '''Construct a dummy feed.'''
+    return {
+            'status': 'ok',
+            'payload': [
+                {
+                    'sources': [
```

Comment:
Needs a `state` entry. The `sources` sub-entry can be an empty object for now, but `index` should be set to `3`

Reply:
Okay, looks good now :)